### DISCUSSION TITLE
An error message about multiple processes

### DISCUSSION THREAD

**Discussion by 0batsy0** [phoebe-project/phoebe2#775] [Questions]
When I use pycharm to run code，I received an error report，It seems to be underlying multiprocess code：
![捕获](https://github.com/phoebe-project/phoebe2/assets/130527283/dbb435d2-6906-4723-9acb-05ddeca4c05b)

**Comment by kecnry** [comment]
Can you try disabling multiprocessing in phoebe (either by calling `phoebe.multiprocessing_off()` immediately after importing or by setting the enviroment variable `PHOEBE_MULTIPROC_NPROCS=0`) and let us know if that fixes the issue?

**Reply by 0batsy0** [reply]
Thank you for your reply.Does a program work normally without multiple processes, which means it can only work in a single process? Is there any way to solve this situation?

**Reply by kecnry** [reply]
Yes, phoebe can work serially or using MPI instead.  The multiprocessing (which is on by default) seems to be incompatible with a few different setups, so the easiest option is to just disable it.
